src/Magnum/OvrIntegration/Context.h

```diff
@@ -53,6 +53,17 @@ struct Error {
 };
 
 /**
+ * @brief Result of @ref Context::detect(int).
+ */
+enum class OvrDetectResultFlag: Byte {
+    ServiceRunning = 1,
+    HmdConnected = 2
+};
+
+typedef Corrade::Containers::EnumSet<OvrDetectResultFlag> OvrDetectResult;
+CORRADE_ENUMSET_OPERATORS(OvrDetectResult)
```

Comment:
Sorry about a late reply, got busy once again ... 

Enum is great, *way* better than a structure of two bools (heh), but I feel like there is an inconsistency in the naming (the enum has `*Flag`, but the enum set doesn't). Elsewhere I was always having either `SomethingFlag` and `SomethingFlags` or tried very hard to invent something else so that is always easy to guess name of the other type. 

Hmm, what about `OvrDetectResult` for the enum and `OvrDetectResults` for the enumset?